content/en/blog/_posts/2022-12-05-Security-Behavior-Analysis/index.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: blog
+title: "All Microservices Are Vulnerable"
+date: 2022-12-05
+slug: security-behavior-analysis
+---
+
+**Author:**
+David Hadas (IBM Research Labs)
+
+As cyber war games continue to intensify in sophistication, organizations deploying cloud services, continue to grow their cyber investments aiming to produce safe and non-vulnerable services. However, the year-by-year growth in cyber investments, does not result in parallel reduction in cyber incidents. Instead, the number of cyber incidents continue to grow annually. Evidently organizations are fighting a battle they are doomed to fail at. No matter how much effort is made to detect and remove cyber weaknesses from deployed services, it seems offenders have the upper hand.
+
+Considering the current spread of offensive tools, sophistication of offensive players, and ever-growing cyber financial gains to offenders, any cyber strategy that relies on constructing a non-vulnerable, weakness-free service in 2023 is clearly too naïve. It seems the only viable strategy is to:
+
+------> **Admit that your services are vulnerable!**
+
+In other words, consciously accept that you will lose any battle aimed to create non-vulnerable services. If your opponents find a single weakness as an entry-point, you lose. Admitting that although your best efforts, all your services are most likely vulnerable is an important first step. Next, we need to discuss what can you do about it...
+
+This post discuss the cyber challenges that Kubernetes users face when deploying microservices with Kubernetes and how to overcome them by performing *"Security-Behavior Analysis"* - that is, by analyzing the behavior of clients and services from a security standpoint. It also points to [Guard](http://knative.dev/security-guard), an open source project offering security-behavior monitoring and control of your microservices deployed on a Kubernetes platform, given that they are presumed vulnerable.
```

Comment:
Is it `guard` or `security guard` ? If both are used interchangeably that's fine but just wanted to check

Reply:
Knative security WG where it is hosted officially calls it security-guard to tie it up with the security WG work vs other work in the Knative community. I am using Guard interchangeably.